README.md

```diff
@@ -35,8 +35,8 @@ paths like `github.com/google/ko/cmd`.
 
 **One of the goals of `ko` is to make containers invisible infrastructure.**
 Simply replace image references in your Kubernetes yaml with the import path for
-your Go binary, and `ko` will handle containerizing and publishing that
-container image as needed.
+your Go binary prefixed with `ko://`, and `ko` will handle containerizing and
```

Comment:
Worth taking this opportunity to add an example import path? (`ko://github.com/user/repo/cmd/server`)

Reply:
Done.